conda/common/serialize/__init__.py

```diff
@@ -60,11 +60,30 @@ def yaml_safe_dump(object, stream=None):
         return ostream.getvalue()
 
 
-def json_load(string):
-    return json.loads(string)
-
-
+deprecated.constant(
+    "25.9",
+    "26.3",
+    "EntityEncoder",
+    CondaJSONEncoder,
+    addendum="Use `conda.common.serialize.json.CondaJSONEncoder` instead.",
+)
+del CondaJSONEncoder
+deprecated.constant(
+    "25.9",
+    "26.3",
+    "json_load",
+    loads,
+    addendum="Use `conda.common.serialize.json.loads(sort_keys=True)` instead.",
+)
+del loads
+
+
+@deprecated(
+    "25.9",
+    "26.3",
+    addendum="Use `conda.common.serialize.json.dumps(sort_keys=True)` instead.",
+)
 def json_dump(object):
-    return json.dumps(
-        object, indent=2, sort_keys=True, separators=(",", ": "), cls=EntityEncoder
```

Comment:
A funny choice from past conda developers, to make only one of them include a space... for me, the most compact version is best.

Reply:
🤷🏼‍♂️